code/game/machinery/firealarm.dm

```diff
@@ -263,6 +263,16 @@ FIRE ALARM
 
 /obj/machinery/firealarm/examine(mob/user)
 	. = ..()
+	switch(buildstage)
+		if(FIRE_ALARM_FRAME)
+			. += "<span class='notice'>It's missing the <i>control board<i>, but the <b>mounting bolts</b> are exposed.</span>"
```

Comment:
probably better to say circuit board here, and maybe change it from `mounting bolts` to just `bolts`